examples/vnet/README.md

```diff
@@ -4,6 +4,16 @@
 
 These examples show you how to build a customized Docker enabled cluster on Microsoft Azure where you can provide your own VNET.
 
+By providing your own VNET, you can change the VNET address space from the default of 10.0.0.0/8 to a smaller subnet, like
+10.1.0.0/16. This may be useful if you have existing VNETs that you would like to peer with, or plan to have them in the future.
+For Kubernetes, a change of the address space only affects the IP addresses assigned to the master and agent nodes. The service
+network of 10.0.0.0/16 is still implemented by the kube-proxy in each host, and the pod network stays as 10.244.0.0/16. This
+means that access to services from another VNET will have to be done via node port through the host addresses, or via an
+internal load balancer (note that currently, internal load balancers need to be created manually).

```

Comment:
Yes, that capability has been added in upstream since this pull request was created. Updated.